applog.go

```diff
@@ -0,0 +1,25 @@
+package mozlogrus
+
+import (
+	"encoding/json"
+
+	"github.com/sirupsen/logrus"
+)
+
+// appLog implements Mozilla logging standard
+type appLog struct {
+	Timestamp  int64
+	Time       string
+	Type       string
+	Logger     string
+	Hostname   string `json:",omitempty"`
+	EnvVersion string
+	Pid        int           `json:",omitempty"`
+	Severity   int           `json:",omitempty"`
+	Fields     logrus.Fields `json:",omitempty"`
```

Comment:
nit: maybe add json tags to all fields, for explicitness?

Reply:
mozlog seems to use Uppercase field names. There's no value to verbosity.